rfcs/InputUnion.md

```diff
@@ -148,6 +148,8 @@ The premise of this RFC - GraphQL should contain a polymorphic Input type.
 
 Any data structure that can be modeled with output type polymorphism should be able to be mirrored with Input polymorphism. Minimal transformation of outputs should be required to send a data structure back as inputs.
 
+* Objection: input types and output types are distinct. Output types support aliases and arguments whereas input types do not. Marking an output field as non-nullable is a non-breaking change, but marking an input field as non-nullable is a breaking change.
```

Comment:
"input types and output types are distinct".. the criteria here is more about the fact that there are data structures that you can model in output types that are impossible to model with input types.

It seems to me that the fact that input and output types are distinct and have different behaviors doesn't really address the intention of this criteria

Reply:
This item currently uses the terms "matches" and "mirrors" indicating that input and output polymorphism should look and feel nearly identical. This objection raises the point that input and output objects don't need to be similar in order to achieve the goal of enabling modelling polymorphic inputs.